policy/subpolicy/daily_schedule_subpolicy_test.go

```diff
@@ -0,0 +1,42 @@
+package subpolicy
+
+import (
+	"testing"
+
+	"github.com/datagovsg/nomad-parametric-autoscaler/resources"
+)
+
+func TestDecodingDailyScheduleSP(t *testing.T) {
```

Comment:
optional: can include tests for invalid body should return error, valid body but invalid params should return error